react/LocationInput.tsx

```diff
@@ -36,7 +41,7 @@ const LocationInput: React.FC<Props> = ({
   useEffect(() => {
     let cancelled = false
 
-    if (data) {
+    if (data?.getAddressFromPostalCode) {
```

Comment:
@lucasecdb looks ok to you? Just making sure that you see that I've refactored LocationInput (basically just added query typings)

Reply:
yep! lgtm